doc/src/quickstart/index.rst

```diff
@@ -239,124 +231,38 @@ Let's start by making a fresh directory for us to work in:
 
 Next we need to run the three main sets of tools:
 
-* :ref:`odin_ii` performs 'synthesis' which converts our behavioural Verilog (``.v`` file) into a circuit netlist (``.blif`` file) consisting of logic equations and FPGA architecture primitives (Flip-Flops, adders etc.),
+* :ref:`Parmys` performs 'synthesis' which converts our behavioural Verilog (``.v`` file) into a circuit netlist (``.blif`` file) consisting of logic equations and FPGA architecture primitives (Flip-Flops, adders etc.),
 * :ref:`ABC` performs 'logic optimization' which simplifies the circuit logic, and 'technology mapping' which converts logic equations into the Look-Up-Tables (LUTs) available on an FPGA, and
```

Comment:
Is this line still correct? Is ABC being used when manually running the VTR flow?

Reply:
it is used when we used synthesize with ODIN.

Reply:
Yes, but these bullet points are in the "Manually Running the VTR Flow" section; not the with ODIN section.

Reply:
Agreed. This should be refactored to make clear you either need to run Odin II + ABC, or parmys (which will internally call ABC itself).